docs/_examples/adapter_and_device_information.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: page
+title: Adapter and Device Information
+order: 10
+---
+
+# Adapter and Device Information
+
+* [Source](https://github.com/dj2/Dusk/tree/main/src/example_01)
+
+This is a simple example to show retrieving information about the
+Adapter and Device.
+
+There are three items needed in order to create the
```

Comment:
Do you explain somewhere what an instance an adapter and a device are semantically?